### DISCUSSION TITLE
Optionally include instructions

### DISCUSSION THREAD

**Discussion by ElinoraWhite** [Azure/PSDocs.Azure#54] [Ideas]
There may be some benefit to allowing custom instructions to be included in template documentation.

```markdown
# Storage Account

Create or update a Storage Account.

<!-- Add custom instructions -->

## Parameters
```

For example:

---

# Storage Account

Create or update a Storage Account.

## Instructions for use

This template can be deployed through our DevOps process using a pull request.
For instructions on how to complete a PR see [about pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).

**Comment by ElinoraWhite** [comment]
@vicperdana Thoughts?

**Reply by vicperdana** [reply]
Haven't received requests re. this feature- but I believe this will be useful for customers who have pre-defined processes and can link them here.
